raiden/tests/scenarios/bf1_basic_functionality.yaml

```diff
@@ -243,5 +245,5 @@ scenario:
             ## To make sure the transactions gets mined in time, 10 additional blocks are added
             ## Monitoring Service only reacts for the second closing because node4 settled to first
             ## channel itself
-            - wait_blocks: 10
+            - wait_blocks: 5
```

Comment:
Are you sure that this is enough? I'd rather wait 5 more blocks here than have scenarios fail because of this.

Reply:
Tested this 3 times already and seems to be enough. Would you like to increase it to give the MS extra time to react?